easybuild/easyconfigs/m/MVAPICH2/MVAPICH2-2.2b-GCC-4.9.3-2.25.eb

```diff
@@ -12,6 +12,12 @@ sources = [SOURCELOWER_TAR_GZ]
 # Let's store the checksum in order to be sure it doesn't suddenly change
 checksums = ['5651e8b7a72d7c77ca68da48f3a5d108']
 
-builddependencies = [('Bison', '3.0.4')]
+builddependencies = [
+    ('Bison', '3.0.4'),
+    ('libibmad', '1.3.12'),
+    ('libibumad', '1.3.10.2'),
```

Comment:
@verdurin any idea how closely these are tied to the IB kernel drivers and `libibverbs`?